WordPress/wordpress-playground
# Move puzzle app to a Playground package

## What is this PR doing?

It moves the [Playground puzzle app](https://github.com/bgrgicak/playground-puzzle) into a Playground package.

## What problem is it solving?

It enables us to deploy the puzzle app to the Playground server.

## How is the problem addressed?

By creating a new Playground package and updating the code to work with Vite.

## Testing Instructions

- Follow the readme instructions to start a dev server
- Setup the [puzzle API using these instructions](https://github.com/WordPress/wordpress-playground/pull/1372)
- [Update this line ](https://github.com/WordPress/wordpress-playground/pull/1385/files#diff-36e03a13c2f651337a0595e14c5aacbcd9e41ed263b271d973d0b0ef6425016fR1)to point to your development API
- Open the app and click on _Build your site_
- Write [one of the action keys](https://github.com/WordPress/wordpress-playground/pull/1385/files#diff-ea60a008cc4118065c2be0703f529a8f2176ad4b38504f94fd68d25550892177R21) like _multisite_ on a piece of paper (text should be clear and bold)
- Scan it
- Confirm that an action was added
- Click on _Take me to my site_
- Confirm that the site load and the actions were applied

**Note let's address design feedback and improvements in separate PRs to unblock sponsors from updating their blueprints**
